sbncode/FlashMatch/FlashPredict.hh

```diff
@@ -148,87 +292,74 @@ private:
   const double fBeamWindowStart, fBeamWindowEnd;
   const double fFlashStart, fFlashEnd;
   const unsigned fTimeBins;
-  const bool fSelectNeutrino, fOnlyPrimaries;
+  const bool fSelectNeutrino;
+  const bool fOnlyCollectionWires;
+  const bool fForceConcurrence;
```

Comment:
Same thing here, some comments to explain what `ForceConcurrence` means would make the code easier to understand.

Reply:
`ForceConcurrence` is a tad too complex to explain on an inline comment. Still I put one elsewhere.